ios/Openid4vpBle/Error/ErrorHandler.swift

```diff
@@ -1,22 +0,0 @@
-import Foundation
```

Comment:
@vharsh (note to self): This file needs to be moved appropriately in `release` branch.

Reply:
This has been now removed from release branch